src/s1ap/main.c

```diff
@@ -415,7 +413,9 @@ accept_sctp(void *data)
 			if (FD_ISSET(sd, &readfds)) {
 				if ((valread = recv_sctp_msg(sd, buffer, SCTP_BUF_SIZE)) <= 0) {
```

Comment:
maybe we should have different handling for recv == 0 and recv < 0 ?

Reply:
man page is silent about = 0 return value. And some google result suggest that even = 0 should be treated as failure.